### DISCUSSION TITLE
Weekly Sync 24th Jan 2022

### DISCUSSION THREAD

**Discussion by matdehaast** [interledger/rafiki#219] [General]
## Updates 

- Brandon
   - Started on adding Open Payments being able to specify deliver amounts or arbitrary asset amount
- Sabine
   - Working on Blog Post
   - Worked on GNAP and OP documents
- Damien
   -  Working on Coil work

## Timelines & Project Scope

- [ ]  CFP opens in March/April would help to have API driven docs available (@matdehaast )
 
## Review/Spec/TODO
- [ ] Explore if we should create a test harness against Rafiki
- [x] @wilsonianb to add issue regarding Web Monetization Webhooks (Time and Amount based rules)
- [ ] @adrianhopebailie @sabinebertram @AlexLakatos to coordinate GNAP work

**Comment by wilsonianb** [comment]
>Web Monetization Webhooks (Time and Amount based rules)

Should rules be Rafiki-wide or per account?

:thinking: Amount based rules would need to either be per asset or per account (or a Rafiki-wide amount would need to denominated in one on asset and converted to others)

**Reply by matdehaast** [reply]
@wilsonianb lets just make this Rafiki wide initially
